systemd/ua.service

```diff
@@ -0,0 +1,20 @@
+# If you are uninterested in the (free for personal use) Ubuntu Advantage
+# services, including security updates after standard EOL and kernel patching
+# without rebooting, then you can safely stop and disable this service:
+# sudo systemctl stop ua.service
+# sudo systemctl disable ua.service
+
+[Unit]
+Description=Ubuntu Advantage Daemon
+Documentation=man:ubuntu-advantage https://github.com/canonical/ubuntu-advantage-client/blob/main/README.md https://ubuntu.com/advantage
+After=network.target network-online.target systemd-networkd.service ua-auto-attach.service
+
+[Service]
+# Xenial and Bionic systemd don't support Type=exec
+Type=simple
```

Comment:
I wonder if using `notify` instead would be feasible here. If so, this would allow more control on failures, possible future service dependencies, etc. This would require checking if we could send notifications through all the versions of python-systemd all the way down to xenial though...

Reply:
From my testing it seems doable! I'll push up a version that does this